### PR TITLE
Revert implementation of unmerged UpdateExecution API changes

### PR DESCRIPTION
**Pull request by katrogan**
Signed-off-by: Katrina Rogan <katroganGH@gmail.com>

# TL;DR
https://github.com/flyteorg/flyteadmin/pull/317 adds a dependency on a specific sha, rather than release of flyteidl which breaks the flyte release process, specifically generating config docs: https://github.com/flyteorg/flyte/actions/runs/1714466031

Until we merge API changes in https://github.com/flyteorg/flyteidl/pull/248, this change reverts their implementation so we can stick to an official idl release

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue
https://github.com/flyteorg/flyte/issues/2045

## Follow-up issue
_NA_

### CODE REVIEW

**Review by EngHabu — APPROVED** [review]
Thank you!

**Comment by katrogan** [comment]
cc @pmahindrakar-oss
